Presentation/Airlines/AirlinesViewModel.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  AirlinesViewModel.swift
+//  AirlinesApplication
+//
+//  Created by HendEl-Mahdy on 19/07/2024.
+//
+
+import Foundation
+import RxSwift
+
+protocol AirlinesProtocol {
+    var airlinesArraySubject: BehaviorSubject<Result<Void, AppError>> { get }
```

Comment:
We can use observable because it is read-only, BehaviorSubject is both an observer and an observable.